ui/app/mainui/AppMain.qml

```diff
@@ -1630,6 +1630,16 @@ Item {
                             }
 
                             onSettingsSubsectionChanged: profileLoader.settingsSubsection = settingsSubsection
+
+                            onLaunchSendModal: {
+                                let ensUsernamesStore = appMain.rootStore.profileSectionStore.ensUsernamesStore
+                                popups.openSendModal(sendType, senderAddress, tokenId, tokenType, tokenAmount, chainId,
+                                                     ensUsernamesStore.getEnsRegisteredAddress() /*recipientAddress*/,
+                                                     SendPopups.Helpers.RecipientAddressObjectType.Address /*recipientType*/,
+                                                     false /*onlyAssets*/,
+                                                     false /*interactive*/,
+                                                     publicKey, ensName)
+                            }
```

Comment:
hmmm so we do end up having multiple function on the Popups.qml file for launching send. Hos do we feel about those being calls to specific Global function instead so that we dont have to pass the information along the whole chain? which is then inline with with Alex and Lukas also mentioned?

Reply:
Functions are more flexible, because you can default the arguments, whereas for signals, you must spell all of them out. Otherwise connecting a signal to another signal is fine for sure